test_result

```diff
@@ -0,0 +1 @@
+{'key': 'test.wav', 'pam_score': 0.01386283989995718}
```

Comment:
We add some specific test scripts for individual models at test/test_{metric}.py. You may consider refer to that to add the test function~